hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnectionManagerBase.java

```diff
@@ -113,14 +119,88 @@ void refreshNetworkStats() {
     }
 
     static class Plane {
-        final ConcurrentHashMap<Address, TcpServerConnection> connectionMap = new ConcurrentHashMap<>(100);
-        final Set<Address> connectionsInProgress = newSetFromMap(new ConcurrentHashMap<>());
         final ConcurrentHashMap<Address, TcpServerConnectionErrorHandler> errorHandlers = new ConcurrentHashMap<>(100);
         final int index;
 
+        private final Set<Address> connectionsInProgress = newSetFromMap(new ConcurrentHashMap<>());
+        private final ConcurrentHashMap<Address, TcpServerConnection> connectionMap = new ConcurrentHashMap<>(100);
+
         Plane(int index) {
             this.index = index;
         }
+
+        TcpServerConnection getConnection(Address address) {
+            return connectionMap.get(address);
+        }
+
+        void putConnection(Address address, TcpServerConnection connection) {
+            connectionMap.put(address, connection);
+        }
+
+        void putConnectionIfAbsent(Address address, TcpServerConnection connection) {
+            connectionMap.putIfAbsent(address, connection);
+        }
+
+        void removeConnection(TcpServerConnection connection) {
+            removeConnectionInProgress(connection.getRemoteAddress());
+            connectionMap.entrySet().removeIf(entry -> entry.getValue().equals(connection));
+        }
+
+        public Set<Address> getAddresses(TcpServerConnection connection) {
```

Comment:
method seems unused, leftover?

Reply:
Ahh... Some of these refactoring are from the hostname related branch I'm also working on, because I accidentaly started working on this on that branch, then moved it over to master. It's used there, I'll remove it here.